as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/reddeer/util/OperateServerTemplate.java

```diff
@@ -144,7 +144,12 @@ public void startServer() {
 	}
 
 	public void restartServer() {
-		serversView.getServer(getServerName()).restart();
+		try {
+			serversView.getServer(getServerName()).restart();
+		} catch (WaitTimeoutExpiredException ex){
```

Comment:
Are you sure with this? This seems to be very specific workaround...

Reply:
We discussed this workaround with @jkopriva and this seems to be a workaround and result will be monitored.